src/ui/server/helpers/should-nullify-loss-payee-financial-uk-data/index.ts

```diff
@@ -1,22 +1,35 @@
 import { objectHasKeysAndValues } from '../object';
+import FIELD_IDS from '../../constants/field-ids/insurance/policy';
 import { ApplicationNominatedLossPayee } from '../../../types';
 
+const {
+  LOSS_PAYEE_DETAILS: { IS_LOCATED_INTERNATIONALLY },
+} = FIELD_IDS;
+
 /**
  * shouldNullifyLossPayeeFinancialUkData
  * Check if we should nullify loss payee financial UK data.
- * If IS_APPOINTED is true and the application has some financial UK data, the charge data should be nullified.
+ * If IS_APPOINTED is true
+ * Or the location is IS_LOCATED_INTERNATIONALLY
+ * and the application has some financial UK data,
+ * the UK data should be nullified.
  * @param {String} isAppointed: Is appointed form field
+ * @param {String} location: Loss payee location form field
  * @param {ApplicationNominatedLossPayee} lossPayeeData: Loss payee data
  * @returns {Boolean}
  */
-const shouldNullifyLossPayeeFinancialUkData = (isAppointed: string, lossPayeeData: ApplicationNominatedLossPayee) => {
+const shouldNullifyLossPayeeFinancialUkData = (isAppointed: string, location: string, lossPayeeData: ApplicationNominatedLossPayee) => {
   const {
     financialUk: { id, ...financialUkFields },
   } = lossPayeeData;
 
-  const hasFinancialDataUk = objectHasKeysAndValues(financialUkFields);
+  const hasData = objectHasKeysAndValues(financialUkFields);
+
+  const isNotAppointed = isAppointed === 'false';
```

Comment:
do we have a helper for this

Reply:
For checking a string with a value of false? No, we could do with one though. We have over 20 instances of false string checks, could do as a tech improvement.

Reply:
Created a ticket for this => https://ukef-dtfs.atlassian.net/browse/EMS-3356